desktop/app/window-handlers/navigation/index.js

```diff
@@ -32,4 +32,14 @@ module.exports = function ( { view } ) {
 			view.webContents.loadURL( webBase + 'stats/day' );
 		}
 	} );
+
+	ipcMain.on( 'title-bar-double-click', () => {
+		if ( process.platform === 'darwin' ) {
+			const action = systemPreferences.getUserDefault( 'AppleActionOnDoubleClick', 'string' );
+			if ( action === 'None' ) return;
+			if ( action === 'Minimize' ) return window.minimize();
+		}
+		if ( window.isMaximized() ) return window.unmaximize();
```

Comment:
Was just testing this and it works well! However I think we'll want to disable this feature on Linux. It seems the browserview doesn't expand with the resized window in this scenario. We can exclude from Linux for the time being and note as an enhancement to investigate later.

Also one general note: this is fine for now but this logic doesn't quite pertain to "navigation" per se. It may be better off in a separate module/file. I'm not sure where, though. 🤔